gmtcn.sh

```diff
@@ -50,30 +51,45 @@ if [ ! -n "${2}" ]; then
     exit
 fi
 
+# system
+sys=$(uname)
+if [ ${sys} = "Linux" ] ; then
+    open_command=xdg-open
```

Comment:
I think `open` should be good. No need to have such a long var name.
```suggestion
    open=xdg-open
```

Reply:
updated